### PR TITLE
Added solution for BooleanExpression

### PR DESCRIPTION
**Pull request by Nokafan**

### CODE REVIEW

File: src/main/java/core/basesyntax/BooleanExpression.java [hunk 1]
```diff
@@ -14,6 +14,19 @@
      * без использования конструкций if-else</p>
      */
     public boolean booleanExpression(boolean a, boolean b, boolean c, boolean d) {
-        return false;
+        int result = 0;
+        if (a) {
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/BooleanExpression.java]: read [checklist](https://mate-academy.github.io/jv-program-fulltime/01_core/checklist/hw_01.html) attentively
- Use only boolean operators without if-else or ternary operator.

Comment (Sofasmile) [reply src/main/java/core/basesyntax/BooleanExpression.java]: read smth about XOR operation and `||` and `&&`

Comment (Nokafan) [reply src/main/java/core/basesyntax/BooleanExpression.java]: Did it! Happy because it wasn't easy for me :(

**Review by Sofasmile — APPROVED** [review]
great job

**Pull request closed without merging**